src/package/components/Request/components/RequestBase/index.js

```diff
@@ -102,12 +102,12 @@ class RequestBase extends Component {
   }
 
   render() {
-    const {children, lazy, placeholder} = this.props
+    const {children, lazy, placeholder, showLoadingBetweenRequests} = this.props
     const {running, error, data, response} = this.state
     if (!children) {
       return null
     }
-    if (running && (lazy || !data)) {
+    if ((running && showLoadingBetweenRequests) || (running && (lazy || !data))) {
```

Comment:
Show placeholder/loading view on the first request, but not on subsequent requests unless `showLoadingBetweenRequests` is on